style-guide/about-style-guide.md

```diff
@@ -0,0 +1,183 @@
+# The style guide
+
+The style guide template itself occasionally includes placeholder text that
```

Comment:
This is just the standard at my workplace, but I'm open to suggestions.

Reply:
Huh, that's interesting. I don't think it's worth changing, but it might be worth discussing if we want to have a project standard for this.

Reply:
If we care about source style for Markdown, then we may want to adopt a tool like [Prettier](https://prettier.io/). It would push the responsibility of choosing a style onto a tool (so we don't have to debate it) and makes keeping to that style a snap (because the tool does it for you).

Opened an issue for this idea: https://github.com/thegooddocsproject/templates/issues/160

Reply:
Thanks for opening that @ddbeck . @Loquacity , I'd definitely be open to changing this once we decide on the line break standard. Line break debates seem to be a constant point of division at any workplace I go to, but I don't personally have a dog in the fight, so I'll conform with whatever we decide.